src/lib/optimizer/optimizer.cpp

```diff
@@ -7,6 +7,7 @@
 #include "strategy/index_scan_rule.hpp"
 #include "strategy/join_detection_rule.hpp"
 #include "strategy/predicate_reordering_rule.hpp"
+#include "strategy/projection_combination_rule.hpp"
 
```

Comment:
Random place for conversation.

This comment https://github.com/hyrise/hyrise/pull/733#discussion_r170472496 is somehow hidden now, shouldn't get lost though.

Reply:
Oh wow, I totally didn't see that one yesterday... Thanks, I'll look into that!